infra_tooling_account/app.py

```diff
@@ -18,12 +18,14 @@
 InfraToolingCommonStack(
     app,
     f"cf-{PROJECT_NAME}-InfraToolingCommonStack-{STAGE_NAME}",
+    tags={"project_name": PROJECT_NAME, "stage_name": STAGE_NAME},
     stage_name=STAGE_NAME,
     project_name=PROJECT_NAME,
 )
 InfraToolingAlertingStack(
     app,
     f"cf-{PROJECT_NAME}-InfraToolingAlertingStack-{STAGE_NAME}",
+    tags={"project_name": PROJECT_NAME, "stage_name": STAGE_NAME},
```

Comment:
Alerting stack now relies on Common stack's output, so we need to make sure Alerting is created ONLY after Common is completed (I bumped into problem like this).

commonStack = InfraToolingCommonStack(
    app,
    f"cf-{PROJECT_NAME}-InfraToolingCommonStack-{STAGE_NAME}",
    tags={"project_name": PROJECT_NAME, "stage_name": STAGE_NAME},
    stage_name=STAGE_NAME,
    project_name=PROJECT_NAME,
)
infraStack = InfraToolingAlertingStack(
    app,
    f"cf-{PROJECT_NAME}-InfraToolingAlertingStack-{STAGE_NAME}",
    tags={"project_name": PROJECT_NAME, "stage_name": STAGE_NAME},
    stage_name=STAGE_NAME,
    project_name=PROJECT_NAME,
)

infraStack.add_dependency(commonStack)

Reply:
Done